NthKey/Model/KeychainEntropyItem.swift

```diff
@@ -1,12 +1,12 @@
 /*
-    Copyright © 2016 Apple Inc. All Rights Reserved.
-    Copyright © 2019 Purple Dunes. Distributed under the MIT software
-    license, see the accompanying file LICENSE.md
-
-    Abstract:
-    A struct for accessing BIP39 entropy Data in the keychain. This entropy
-    can be converted to a 12-24 word BIP39 mnemonic.
-*/
+ Copyright © 2016 Apple Inc. All Rights Reserved.
+ Copyright © 2019 Purple Dunes. Distributed under the MIT software
+ license, see the accompanying file LICENSE.md
+ 
+ Abstract:
+ A struct for accessing BIP39 entropy Data in the keychain. This entropy
+ can be converted to a 12-24 word BIP39 mnemonic.
+ */
```

Comment:
These whitespace changes clutter up the commit, so would be good to put those in a separate commit (maybe 1 to cover all files). Are you using a tool for this?

Reply:
This is xcode default indentation tool, I'll separate that commit.